### PR TITLE
[ADD] contact_employment_info: Living situation, academic record, sco…

### PR DESCRIPTION
**Pull request by Berezi**
…larship record , expectation and employment situation in res partner.

### CODE REVIEW

**Comment by Berezi** [comment]
@alfredoavanzosc @oihane @anajuaristi @Daniel-CA

**Review by anajuaristi — APPROVED** [review]

File: contact_employment_info/models/res_partner.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    _inherit = 'res.partner'
+
+    living_situation_id = fields.Selection(
+        [('separate', 'Separate'), ('coexistence', 'Coexistence')],
+        string='Living situation')
+    can_contact_with = fields.Boolean(
+        string='Can contact with?', default=False)
+    academic_record = fields.Boolean(
+        string='Academic record', default=False)
+    scolarship_request = fields.Boolean(
```
Comment (anajuaristi) [inline contact_employment_info/models/res_partner.py]: scholarship (Con h intercalada), en todos los sitios que salga en el módulo

File: contact_employment_info/models/res_partner.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
+# Copyright 2021 Berezi - Iker - AvanzOSC
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+from odoo import models, fields
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    living_situation_id = fields.Selection(
```
Comment (anajuaristi) [inline contact_employment_info/models/res_partner.py]: quita _id porque es un selection
deja living_situation

**Comment by Berezi** [comment]
@alfredoavanzosc @anajuaristi @oihane @Daniel-CA review please.

**Review by alfredoavanzosc — APPROVED** [review]
:+1

**Pull request merged**